#Standard header
Header header

#######################
# Debug 0-23
#######################
uint16 register_00
uint16 register_01
uint16 register_02
uint16 register_03
uint16 register_04
uint16 register_05
uint16 register_06
uint16 register_07
uint16 register_08
uint16 register_09
uint16 register_10
uint16 register_11
uint16 register_12
uint16 register_13
uint16 register_14
uint16 register_15
uint16 register_16
uint16 register_17
uint16 register_18
uint16 register_19
uint16 register_20
uint16 register_21
uint16 register_22
uint16 register_23

